packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -985,20 +986,6 @@ class WebpackCLI implements IWebpackCLI {
           "Sets process.env.NODE_ENV to the specified value. (Currently an alias for `--node-env`).",
         helpLevel: "verbose",
       },
-
-      // Adding more plugins
-      {
-        name: "analyze",
-        configs: [
-          {
-            type: "enum",
-            values: [true],
-          },
-        ],
-        multiple: false,
-        description: "It invokes webpack-bundle-analyzer plugin to get bundle information.",
-        helpLevel: "minimum",
-      },
```

Comment:
Why was this removed?

Reply:
I was thinking of a custom plugin, so was planning on changing stuff accordingly. But got it, will start working on the last action.